packages/renderer/src/lib/extensions/CatalogExtensionList.svelte

```diff
@@ -6,6 +6,8 @@ import type { CatalogExtensionInfoUI } from './catalog-extension-info-ui';
 import CatalogExtension from './CatalogExtension.svelte';
 
 export let catalogExtensions: CatalogExtensionInfoUI[];
+export let title: string = 'Available extensions';
+export let hideIfEmpty: boolean = false;
```

Comment:
`hideEmpty` is probably enough, I am not a big fan of having conditions in the naming,

other ideas would be `showEmptyScreen`, `displayEmptyScreen` or having a slot, and being empty by default

Reply:
changed